README.md

```diff
@@ -1,31 +1,61 @@
+
+We look to improve upon this preliminary model in the future. 
+We aim to generate further data which includes the heterozygous genotype (*NF1+/-*).
+AS well, we plan to apply an improved model to large-scale drug screens to capture candidate drugs that make NF1 patient cells look healthy.
+
+**NOTE:** All image analysis and image-based profiling pipelines can be found in the [nf1_cellpainting_data](https://github.com/WayScience/NF1_SchwannCell_data_analysis) repository.
+
 ## Repository Structure
+
 This analysis is categorized as follows:
 
 | Module | Purpose | Description |
 | :---- | :----- | :---------- |
-| [0.data_analysis](./0.data_analysis/) | Analyze NF1 Data | We find interesting patterns in the NF1 cell morphology data. |
-| [1.train_models](./1.train_models/) | Train NF1 Models | Optimize the NF1 model by training multiple models with a random search. |
-| [2.evaluate_models](./2.evaluate_models/) | Evaluate Final NF1 Model | After training the final NF1 model we evaluate model performance and model feature importances. |
-| [3.figures](./3.figures/) | Generate Manuscript Figures | We interpret the results from the model by generating figures to include in the manuscript. |
+| [0.data_analysis](./0.data_analysis/) | Preliminary analysis of NF1 data | We find interesting patterns in the NF1 cell morphology data. |
+| [1.train_models](./1.train_models/) | Train NF1 models | Optimize the NF1 model by training multiple models with a random search. |
+| [2.evaluate_models](./2.evaluate_models/) | Evaluate final NF1 model | After training the final NF1 model, we extract the model performance and model feature importances. |
+| [3.figures](./3.figures/) | Generate manuscript figures | We interpret the results from the model by generating figures to include in the manuscript. |
+
+## Steps to reproduce
 
-## Steps to Reproduce
 ### Step 1. Clone the analysis repo
+
 ```sh
 git clone https://github.com/WayScience/NF1_SchwannCell_data_analysis.git
 ```
 
 ### Step 2. Change the current path to the repo path
 
 ### Step 3. Populate the nf1_data_repo folder from the NF1_SchwannCell_data repo
+
 ```sh
 git submodule update --init --recursive
 ```
 
-### Step 4. Create the conda environment
+### Step 4. Create the conda environments
+
+There are two different environments used in this analysis:
+
+1. [environment.yml](./environment.yml): A Python-based environment meant for performing model training, evaluation, and image montage generation.
+2. [figure_environment.yml](./figure_environment.yml): An R-based environment meant for generating figures.
```

Comment:
Could also mention which modules these environment apply to

Reply:
I can see this! It gets a bit tricky since the figures module uses both. I think this is the most simple way of explaining which environments go with which data.